app/models/response_choice.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: response_choices
+#
+#  id                                   :integer          not null, primary key
+#  question_id                          :integer
+#  key                                  :string
+#  text                                 :text
+#  created_at                           :datetime         not null
+#  updated_at                           :datetime         not null
+#
+
+class ResponseChoice < ActiveRecord::Base
+  belongs_to :question
+  has_many :question_orders
+  validates :key, presence: true
```

Comment:
how are we ordering the `response_choices` for a question? we would probably need to use the keys to order?